celery/canvas.py

```diff
@@ -1203,7 +1203,10 @@ def freeze(self, _id=None, group_id=None, chord=None,
             self.tasks = group(self.tasks, app=self.app)
         header_result = self.tasks.freeze(
             parent_id=parent_id, root_id=root_id, chord=self.body)
-        bodyres = self.body.freeze(_id, root_id=root_id)
+
+        bodyres = self.body.freeze(
```

Comment:
better name body_result?

Reply:
I didn't changed any existing code, should I clean it up ?

Reply:
you could try